<launch>
  <node pkg="sound_play" name="soundplay" type="soundplay_node.py" output="screen">
  </node>
  <node pkg="destination" name="user_input" type="user_input.py" output="screen">
  </node>
  <node pkg="destination" name="marker" type="marker.py" output="screen">
  </node>
  <node pkg="destination" name="stop" type="stop.py" output="screen">
  </node>
  <node pkg="destination" name="Main_Control" type="Stage_1_Test.py" output="screen">
  </node>
</launch>
